### PR TITLE
Feature/RUN-3909 missing api functions for .29 compatibility

### PR DESCRIPTION
**Pull request by rdepena**
Missing API functions:
System.downloadPreloadScripts
Window.showDeveloperTools

Along with some changes to the build output to prepare for definitely typed integration.

### CODE REVIEW

File: src/api/window/window.ts [hunk 1]
```diff
@@ -887,6 +887,15 @@ export class _Window extends Base {
...
 
+    /**
+     * Shows the Chromium Developer Tools
+     * @return {Promise.<void>}
+     * @tutorial Window.showDeveloperTools
+     */
+    public showDeveloperTools(): Promise<void> {
+        return this.wire.sendAction('show-developer-tools', this.identity).then(() => undefined);
+    }
+
```
Comment (licui3936) [inline src/api/window/window.ts]: This is System method, not window.

Comment (rdepena) [reply src/api/window/window.ts]: I know, want to move to Window as it is more natural. 

`win.showDeveloperTools();`
versus 
`fin.System.showDeveloperTools('uuid', 'name');`

Comment (licui3936) [reply src/api/window/window.ts]: i see. That means core needs to update too, right?

Comment (rdepena) [reply src/api/window/window.ts]: The core would need no changes.

**Pull request merged**